package.json

```diff
@@ -65,31 +65,15 @@
 			"tsx"
 		],
 		"rules": {
+			"import/no-unresolved": "off",
 			"import/no-unassigned-import": "off",
-			"no-unused-vars": [
-				2,
-				{
-					"varsIgnorePattern": "^React$"
-				}
-			]
+			"@typescript-eslint/explicit-function-return-type": "off",
+			"@typescript-eslint/no-type-alias": "off",
```

Comment:
I'm really struggling to figure out what options to set for this rule. See https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-type-alias.md

There's no clear answer when to use `interface` or `type`...

https://stackoverflow.com/questions/37233735/typescript-interfaces-vs-types
https://medium.com/@martin_hotell/interface-vs-type-alias-in-typescript-2-7-2a8f1777af4c

Feedback wanted.

Reply:
https://github.com/typescript-eslint/typescript-eslint/issues/142

Reply:
In regards to a type vs. Interface, a type is just an alias.

The implications of that are that you can't use declaration  merging with a type that is defined with the "type" keyword. So if you expose it in your declaration files, devs won't be able to enhance that particular type. So in that case prefer an interface. 

See https://twitter.com/nickytonline/status/1114351742945247234